README

```diff
@@ -82,8 +82,15 @@ Setup
 -----
 
 A sample lighttd.conf file is provided, which can be used to test. To test
-CGImap with lighttpd use the supplied config file with lighttpd and run cgimap
-on the command line with a command like
+CGImap with lighttpd you will need to install lighttpd:
+
+	sudo apt-get install lighttpd
+
+Edit the supplied lighttpd.config file to include your CGImap path and run it with the lighttpd like
+
+	/usr/sbin/lighttpd -f lighttpd.conf
+
+Run CGImap on the command line with a command like
```

Comment:
Yes, FastCGI handles starting the binary, but I thought we are starting the binary for debugging?